src/en/controllers-creating.md

```diff
@@ -75,5 +75,5 @@ how the list denotes default regions for each. To specify a different
 region during controller creation, use:
 
 ```bash
-juju bootstrap mycontroller aws/us-west-2
+juju bootstrap aws/us-west-2 mycontroller
 ```
```

Comment:
This is an excellent place to mention dropping the explicit name, because the automatic name reflects the actual region